apps/portal/src/content/docs/components/alert.mdx

```diff
@@ -60,12 +109,46 @@ The `Container` component serves as the main container for all alert elements.
         "You can pass in your `Alert.Title` and `Alert.Description` as children",
       required: true,
       value: "ReactNode",
+      defaultValue: "",
```

Comment:
Not a problem, but `defaultValue` can be omitted for required props.

Reply:
Fixed